notebooks/source/bayesian_regression.ipynb

```diff
@@ -1925,7 +1925,7 @@
    "source": [
     "## Regression Model with Measurement Error\n",
     "\n",
-    "Note that in our previous models, each data point influences the regression line equally. Is this well justified? We will build on the previous model to incorporate measurement error given by `Divorce SE` variable in the dataset. Incorporating measurement noise will be useful in ensuring that observations that have higher confidence (i.e. lower measurement noise) have a greater impact on the regression line. On the other hand, this will also help us better model outliers with high measurement errors. For more details on modeling errors due to measurement noise, refer to Chapter 15 of [[1](#References)].\n",
+    "Note that in our previous models, each data point influences the regression line equally. Is this well justified? We will build on the previous model to incorporate measurement error given by `Divorce SE` variable in the dataset. Incorporating measurement noise will be useful in ensuring that observations that have higher confidence (i.e. lower measurement noise) have a greater impact on the regression line. On the other hand, this will also help us better model outliers with high measurement errors. For more details on modeling errors due to measurement noise, refer to Chapter 14 of [[1](#References)].\n",
```

Comment:
typo: chapter 15 -> chapter 14 (according to this [table of contents](https://www.amazon.com/Statistical-Rethinking-Bayesian-Examples-Chapman/dp/1482253445#reader_1482253445))